### DISCUSSION TITLE
cursor flickering / blinking on message popup

### DISCUSSION THREAD

**Discussion by tkkcc** [folke/noice.nvim#743] [General]
When i search some non-exist word and press enter, an error message popup. Then my cursor become flickering for several seconds. I have this issue in LazyVim default config (even at startup), then I try to make config from kickstart, I got this problem too. 

The cause is that `noice.nvim`, with suggested config in readme, use `nvim-notify` to display message. `nvim-notify` will do animation, and result in cursor flickering in my case: alacritty+tmux or kitty+zellij both. But without tmux/zellij, it's fine.

Solution is to disable nvim-notify's animation or use the mini fallback.

https://github.com/folke/noice.nvim/assets/17373509/a8e373a0-905f-46a3-bfc2-002ba9c6cc5d

**Comment by Sam-programs** [comment]
This is an `nvim-notify` issue, you'll probably get more help by opening an issue [there](https://github.com/rcarriga/nvim-notify/issues).

**Comment by Rewwoken** [comment]
I believe it is connected to `nvim-notify` update rate.

Adjusting the fps property helped me:
```lua
notify.setup({
	fps = 144, -- your monitor refresh rate
})
```
Keep in mind that this may affect the GPU performance.
